specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -1270,6 +1270,56 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateIdleShutdownSetting": {
+        "post": {
+          "tags": [
+              "MachineLearningComputes"
+          ],
+          "operationId": "Compute_UpdateIdleShutdownSetting",
+          "description": "Updates the idle shutdown setting of a compute instance.",
+          "x-ms-examples": {
+              "Update idle shutdown setting of ComputeInstance": {
+                "$ref": "./examples/Compute/updateIdleShutdownSetting.json"
+              }
+          },
+          "parameters": [
+              {
+                "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+              },
+              {
+                "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+              },
+              {
+                "$ref": "#/parameters/WorkspaceNameParameter"
+              },
+              {
+                "$ref": "#/parameters/ComputeNameParameter"
+              },
+              {
+                "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+              },
+              {
+                "name": "parameters",
```

Comment:
Is this a required input for this api? If yes, please add required true property too

Reply:
It is. Added property.